### PR TITLE
Link to docs.

### PR DESCRIPTION
**Pull request by homotechsual**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,15 @@
 CiviCRM Drupal 8 Module
 =======================
 
-This is a beta version of the integration module required to allow CiviCRM and 
+This is a the integration module required to allow CiviCRM and
```
Comment (demeritcowboy) [inline README.md]: Small typo: `a the`. Otherwise agree with all this.

**Comment by demeritcowboy** [comment]
If you're making an update for the typo anyway, do you want to remove the make-it-happen reference on line 49/27?

**Comment by homotechsual** [comment]
Will do :-)

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

MICHAEL O'TOOLE
Managing Director
 T 029 2009 7097
 M 07769 901541
 E mikey@mjco.uk
WWW.MJCO.UK
54 Florentia Street
Cardiff, CF24 4PF
This email was sent whilst on-the-go! Please excuse any unintentional errors or brevity!
 We don't expect you to read a page of terms and conditions all we ask is that you let us know if we sent this to the wrong person and we'd like you to know that nothing in this email is intended to form a binding legal contract. 
 
 MJCO LTD is a company registered in Wales. Registered Number: 08483853. ​Registered Office: 54 Florentia Street, Cardiff CF24 4PF.

**Comment by homotechsual** [comment]
@demeritcowboy can you re-review :-)

**Comment by demeritcowboy** [comment]
Looks good to me!

**Comment by homotechsual** [comment]
@seamuslee001 can you merge this if you're also happy with the changes :-) <3

**Pull request merged**